src/intersections/rayline.jl

```diff
@@ -10,22 +10,26 @@ The intersection type can be one of four types:
 3. overlap of line and ray (OverlappingRayLine -> Ray)
 4. do not overlap nor intersect (NoIntersection)
 =#
-function intersection(f, r1::Ray{N,T}, l1::Line{N,T}) where {N,T}
-  a, b = r1(0), r1(1)
-  c, d = l1(0), l1(1)
+function intersection(f, ray::Ray{N,T}, line::Line{N,T}) where {N,T}
+  a, b = ray(0), ray(1)
+  c, d = line(0), line(1)
 
-  λ₁, λ₂, r, rₐ = intersectparameters(a, b, c, d)
+  # use a normalized point for ray parameter λ₁
+  len1 = norm(b - a)
```

Comment:
Ok, now that we have better names for the arguments of this function, we need to remove the suffix "1" from variable names because it is not helpful. We can rename `len1` to simply `l` for example.

Reply:
Or if you really want to mean the length of the first argument, then l₁ would be more aligned with the code style.

Reply:
in the context of all intersection functions I would prevent to name it `l` or `l₁`, simply because in other intersections files and tests we use `l₁` for lines and changing all tests to line is not the preferred way.

What do you think about `len₁`?

Reply:
I think we need to be consistent in the src directory and make sure that all intersection algorithms follow the same naming convention for function arguments with explicit line, ray, seg. Regarding the test directory, I think we can call l1, l2, ... without issues.